asgs_main.sh

```diff
@@ -1312,6 +1312,18 @@ monitorJobs()
       postScenarioStatus
    fi
    sleep 30 # give buffers a chance to flush to the filesystem
+   if [[ $JOBTYPE == "padcirc" || $JOBTYPE == "padcswan" ]]; then
+      if [[ $QUALITYCONTROL != "off" ]]; then
+         logMessage "$ENSTORM_TEMP: $THIS: Running quality control checks."
+         if [[ ! -e $QUALITYCONTROL ]]; then
+            warn "$ENSTORM_TEMP: $THIS: The quality control script '$QUALITYCONTROL', specified by the QUALITYCONTROL parameter, was not found."
```

Comment:
```suggestion
            warn "$ENSTORM_TEMP: $THIS: The quality control script '$QUALITYCONTROL', specified by the QUALITYCONTROL parameter, was not found or is not executable."
```

Reply:
this change goes with the one right before it (`-x`)

Reply:
Fixed now thank you!